README.md

```diff
@@ -121,6 +127,12 @@ If you try to archive the app and it fails, please do the following:
    <uses-permission android:name="android.permission.CAMERA" tools:node="remove"/>
    ```
 
+3. If your app will use the `OTPublisher.setVideoFilters()` method, you need to include the following in your app/build.gradle file:
```

Comment:
To confirm, setVideoFilters() is the name used in react code for setVideoTransformers()?